tests/test_type_inference/test_classdef.py

```diff
@@ -41,7 +41,8 @@ def test_classdef_method_call():
     module, inferer = cs._parse_text(program)
     attribute_node = list(module.nodes_of_class(astroid.Attribute))[1]
     expected_rtype = attribute_node.parent.type_constraints.type
-    assert attribute_node.type_constraints.type.__args__[-1] == expected_rtype
+    actual_rtype = inferer.type_constraints.lookup_concrete(attribute_node.type_constraints.type.__args__[-1])
```

Comment:
lookup_type returns the type of a variable in the closest enclosing scope given a Node and a name representing the identifier name; in this case we are doing a lookup of a type variable to see if that type variable has been properly unified in the environment.